cinema_service/settings.py

```diff
@@ -20,9 +22,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
It is a security issue to have the DEBUG variable set to True in a production environment. It is recommended to use an environment variable for this setting, similar to how you have done with the SECRET_KEY and the database settings.